tsdb/shard.go

```diff
@@ -173,6 +181,8 @@ func (s *Shard) Open() error {
 		}
 		s.logger.Printf("%s database index loaded in %s", s.path, time.Now().Sub(start))
 
+		go s.monitorSize()
```

Comment:
Just this stat right now.  I agree that we need to have a polling model.   Unfortunately, we need to rework the monitoring framework quite a bit to allow it it pull stats like this that are too complicated to track in real time.